exporters/stats/stackdriver/src/test/java/io/opencensus/exporter/stats/stackdriver/StackdriverStatsExporterTest.java

```diff
@@ -165,8 +165,9 @@ public void registerViewAndExport() throws IOException {
       StackdriverStatsExporter.registerView(view);
 
       verify(mockStub, times(1)).createMetricDescriptorCallable();
-      // The timeout for verifying createTimeSeries needs to match the export interval of exporter.
-      verify(mockStub, timeout(1000).times(1)).createTimeSeriesCallable();
+      // This verification is slower on some machine. Relax the timeout be greater than the export
+      // interval to make this more steady.
+      verify(mockStub, timeout(2000).times(1)).createTimeSeriesCallable();
```

Comment:
It would help to give more detail about what this line is testing, such as "createTimeSeriesCallable() should be called once per second.  This test uses a timeout that is longer than the export interval in order to decrease the chance of failures due to timing."

Reply:
Sure, documented.